typescript/phoenix-sdk/src/market.ts

```diff
@@ -117,9 +139,15 @@ export class Market {
         ...marketData.header.baseParams,
       },
     });
-    const quoteTokenConfig = allTokens.find(
+    const quoteConfig = allTokens.find(
```

Comment:
I don't really like the default token, imo there shouldn't be a case where we'd want a token obj with all undefined properties. If we can find a token we should throw an error because it means our config file is lacking / incorrect